### PR TITLE
[ui] Implement basic workspace area

### PR DESCRIPTION
**Pull request by evamillan**
Adds the `WorkSpace` component, which displays a set of individual cards. The individuals can be saved to the work space by clicking the pin button on `IndividualEntry` or dragging and dropping the table row over the space. The work space can be cleared by clicking a button.

### CODE REVIEW

File: ui/src/App.vue [hunk 1]
```diff
@@ -34,26 +34,59 @@
...
+    },
+    async addSavedIndividual(uuid) {
+      const isSaved = this.savedIndividuals.find(individual => {
+        return individual.identities[0].uuid === uuid;
+      });
+      if (isSaved) {
+        this.snackbar = true;
+        return;
+      }
+      const response = await getProfileByUuid(this.$apollo, uuid);
```
Comment (sduenas) [inline ui/src/App.vue:85]: Is this needed? Can't we reuse the information that is already on the entry? The less we visit the server, the better. Although, if it's too complicated, we can leave it this way.

File: ui/src/components/WorkSpace.stories.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+import { storiesOf } from "@storybook/vue";
```
Comment (sduenas) [inline ui/src/components/WorkSpace.stories.js:1]: Would be possible to add a test to simulate the drag and drop?

**Review by sduenas — CHANGES_REQUESTED** [review]
Please check my comments. We can discuss about them.

File: ui/src/App.vue (hunk 1 above)
Comment (evamillan) [reply ui/src/App.vue]: I had to make a few changes and move the logic around a bit for it to work without the request, but it's better not to visit the server a second time for the same info.

**Comment by evamillan** [comment]
I updated the PR with the changes you requested.

File: ui/src/components/WorkSpace.stories.js (hunk 2 above)
Comment (sduenas) [reply ui/src/components/WorkSpace.stories.js:1]: Great. I see it working.

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**